### ISSUE TITLE
XenForo password is also retain with this Importer?

### ISSUE THREAD

**Issue by RoldanLT** [akhoury/nodebb-plugin-import#204]
I mean, user's don't need to reset their password after I import my XenForo Forum data?
Thanks!

**Comment by akhoury** [comment]
no. it's a little of a pain to retain the passwords, you basically need to still use Xenforo as an oauth service, or write another one that uses the same hashing mechanism and compare agaisnt the same database.

see https://github.com/akhoury/nodebb-plugin-import/issues/64
and this old post https://community.nodebb.org/topic/2688/after-migration-passwords/13
